src/components/Results.vue

```diff
@@ -45,6 +46,9 @@ export default {
     };
   },
   computed: {
+    ...mapGetters({
+      userPrediction: 'getUserPrediction',
```

Comment:
To nie zadziała jako że informacja o wynikach (`resultStats`) może być brana albo ze store albo wyciągana z URLa -w przypadku gdy ktoś kliknie w urla z wynikiem:
https://github.com/fajnie-ze-wiesz/fzw-webapp/blob/e6f2b047ed291c219e8987091466f0a39c7adf6b/src/components/Results.vue#L48-L57

Więc tak naprawdę wszystko powinno być brane stamtąd. Przechowujemy już taką informację w resultStats:
https://github.com/fajnie-ze-wiesz/fzw-webapp/blob/d8db8b2bc26b73e7d4a746c978b87bee0b0abd0a/src/data/result_stats.js#L11


 ale nie ma gettera który to wyciąga, i trzeba by go dopisać w tamtym miejscu (można go nazwać `getCorrectnessEstimate`) i użyć tutaj analogicznie jak jest to tutaj, np:
https://github.com/fajnie-ze-wiesz/fzw-webapp/blob/e6f2b047ed291c219e8987091466f0a39c7adf6b/src/components/Results.vue#L69-L78

Następnie `correctnessEstimate` użyć tam gdzie jest `XXX%` wyżej w kodzie.